terraform/aks/config/development_aks.tfvars.json

```diff
@@ -1,8 +1,10 @@
 {
   "environment": "development",
-  "key_vault_name": "s189t01-gitapi-dv-kv",
-  "key_vault_app_secret_name": "GITAPI-APP-SECRETS-DEVELOPMENT",
-  "key_vault_infra_secret_name": "GIT-INFRA-SECRETS-DEVELOPMENT",
+  "file_environment": "dev",
+  "app_key_vault_name": "s189t01-gitapi-dv-app-kv",
+  "infra_key_vault_name": "s189t01-gitapi-dv-inf-kv",
+  "key_vault_app_secret_name": "API-KEYS",
+  "key_vault_infra_secret_name": "INFRA-KEYS",
```

Comment:
If you've got two key vaults (one for the application and one for infrastructure) you shouldn't need a single secret name as the secrets can be added directly to the key vault and extracted using the secrets module:

```tf 
module "infrastructure_secrets" {
  source = "git::https://github.com/DFE-Digital/terraform-modules.git//aks/secrets?ref=stable"

  azure_resource_prefix = var.azure_resource_prefix
  service_short         = var.service_short
  config_short          = var.config_short
  key_vault_short       = "inf"
}

# example: module.secrets.map.STATUSCAKE-TOKEN
```